content/300-accelerate/600-faq.mdx

```diff
@@ -96,4 +96,20 @@ Even without using Accelerate's global cache, you can still greatly benefit from
 
 ## Can I use Accelerate with other ORMs/query builders/drivers?
 
-No. We currently do not have any plans for supporting other ORMs/query builders or drivers. However, if you're interested in support for other libraries, feel free to reach out and let us know in our [Discord](https://pris.ly/discord) community in the `#accelerate-feedback` channel.
+No. We currently do not have any plans for supporting other ORMs/query builders or drivers. However, if you're interested in support for other libraries, feel free to reach out and let us know in our [Discord](https://pris.ly/discord) community in the `#help-and-questions` channel.
+
+## What is the maximum allowable value for the `ttl` parameter when configuring cacheStrategy?
```

Comment:
allowable or allowed? 🤔

Reply:
also: `cacheStrategy` should be in backticks, no?

```suggestion
## What is the maximum allowable value for the `ttl` parameter when configuring `cacheStrategy`?
```

Reply:
Yes 🙏 !

Reply:
Allowed makes more sense 🧠 🙏